app/user/rest.py

```diff
@@ -178,6 +178,13 @@ def update_user_attribute(user_id):
 
         send_notification_to_queue(saved_notification, False, queue=QueueNames.NOTIFY)
 
+    if current_app.config["FF_SALESFORCE_CONTACT"]:
+        try:
+            updated_user = get_user_by_id(user_id=user_id)
```

Comment:
We do have the updated user data from the request without having to make another db call. But making the db query is less complicated and I don't think this will create any performance issues since we wouldn't have a high volume of requests to this endpoint. 👍

Reply:
Yeah, I waffled on this a bit as well since it's a bit wasteful to re-query the database when we have the data in a dict.